docs/guides/capacity-planning.md

```diff
@@ -0,0 +1,56 @@
+--
+title: "Capacity Planning"
+linkTitle: "Capacity Planning"
+weight: 104
+slug: schemas
+---
+
+
+You will want to estimate how many nodes are required, how many of
+each component to run, and how much storage space will be required.
+In practice, these will vary greatly depending on the metrics being
+sent to Cortex.
+
+Some key parameters are:
+
+ 1. The number of active series. If you have Prometheus already you
+ can query `prometheus_tsdb_head_series` to see this number.
+ 2. Sampling rate, e.g. a new sample for each series every 15
```

Comment:
May be worth to mention this is the Prometheus `scrape_interval` (and link it to https://prometheus.io/docs/prometheus/latest/configuration/configuration/).